utilities/backupable/backupable_db.cc

```diff
@@ -1381,6 +1423,21 @@ Status BackupEngineImpl::CopyOrCreateFile(
   if (s.ok()) {
     s = dest_writer->Close();
   }
+  if (s.ok() && UseNewNaming()) {
+    // When copying SST files and using db_session_id in the name,
+    // try to get DB identities
+    // Note that when CopyOrCreateFile() is called while restoring, this block
+    // of code is not executed as for now we do not use session_id to verify
+    // the restored file
+    if (!src.empty()) {
+      // copying
+      bool isSst = (src.length() > 4 && src.rfind(".sst") == src.length() - 4);
+      if (isSst) {
+        // SST file
+        s = GetFileDbIdentities(src_env, src, db_id, db_session_id);
```

Comment:
We need to support existing SST files without db session identifiers. I recommend they continue to use the old naming scheme. (Benefit: no duplicating existing SST files on incremental backup after upgrade to this version.)

Technically, we should even allow really old SST files that are missing table properties.

Reply:
Right, I shouldn't assume all the old tables have been backed up using the old naming scheme and then switch to the new naming scheme.